SocialCareCaseViewerApi.Tests/V1/Controllers/CaseStatusControllerTests.cs

```diff
@@ -41,7 +39,7 @@ public void ListCaseStatuses404WithCorrectErrorMessageWhenPersonIsNotFound()
         {
             _mockCaseStatusesUseCase.Setup(x => x.ExecuteGet(It.IsAny<long>())).Throws(new GetCaseStatusesException("Person not found"));
 
-            var response = _statusTypeController.ListCaseStatuses(123456789) as NotFoundObjectResult;
+            var response = _caseStatusController.ListCaseStatuses(123456789) as NotFoundObjectResult;
```

Comment:
This needs to be as ObjectResult. If our API isn't returning a result with a 404 status code this test will still pass as that StatusCode is cast to 404. (I also question if 404 is the correct status code)